portal-web/test/functional/com/liferay/portalweb/paths/portlet/announcements/Announcements.path

```diff
@@ -40,6 +40,11 @@
 	<td>//select[contains(@id,'AnnouncementsPortlet') and contains(@id,'priority')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>AD_TITLE_AND_CONTENT</td>
+	<td>//div[contains(@class,'border-neutral')]//h6[contains(text(),'${key_titleAD}')]/following-sibling::div//p[text()='${key_contentAD}']</td>
+	<td></td>
+</tr>
```

Comment:
Hi @marinaasant0s, there are some fixes that you could do on this path:

- Please, replace the 'h6' attribute with a '*'. This makes the path less fragile to possible styling updates.
- Rename the path name to: 'ANNOUNCEMENT_TITLE_AND_CONTENT'. I think that this name is simple and direct.
- Rename the path variables to. 'key_announcementTitle' and 'key_announcementContent'. Just to be more clear.

What do you think?